lib/libutee/tee_api_objects.c

```diff
@@ -532,8 +532,8 @@ TEE_Result TEE_RenamePersistentObject(TEE_ObjectHandle object,
 	return res;
 }
 
-TEE_Result TEE_AllocatePersistentObjectEnumerator(TEE_ObjectEnumHandle *
-						  objectEnumerator)
+TEE_Result TEE_AllocatePersistentObjectEnumerator(TEE_ObjectEnumHandle
```

Comment:
This looks just as bad as the original in my opinion. If it **has** be changed do it like:
```
TEE_Result TEE_AllocatePersistentObjectEnumerator(
			TEE_ObjectEnumHandle *objectEnumerator)
```
(TEE_ObjectEnumHandle is indented with 3 tabs)